DEVELOPMENT.md

```diff
@@ -3,7 +3,7 @@
 ## Dev Env Setup
 
```

Comment:
Added. I do not think that ASDF installation should be part of `make setup` as not everyone (for example me) use ASDF for managing their development environment. That would cause needless headache for such people.